tests/transforms/test_insert_ops.py

```diff
@@ -91,7 +91,11 @@ def test_start(self):
             for o1, o2 in zip(tape.operations, tape_exp.operations)
         )
         assert len(tape.measurements) == 1
-        assert tape.observables[0].name == ["PauliZ", "PauliZ"]
+        assert (
+            tape.observables[0].name == "Prod"
+            if qml.operation.active_new_opmath()
+            else ["PauliZ", "PauliZ"]
+        )
```

Comment:
@astralcai are those the ones you mean?
@PietropaoloFrisoni I guess those tests here need the fixture `@pytest.mark.usefixtures("use_legacy_and_new_opmath")` for the two cases with new opmath active and not to be hit

Reply:
Oh is this fixture new?

Reply:
Right now I am trying to fix some tests that skipped since they cause an error only with CI (those are skipped on my laptop). If necessary, I'll create a separate story for those

Reply:
My understanding is that new opmath is enabled by default, so without a test cases fixed with `use_legacy_opmath`, legacy behaviour won't be tested anymore.

Reply:
Got them! Now all tests pass on CI as well :fire: :fire: :fire:

I'll let you decide if we want to double the test or not with `@pytest.mark.usefixtures("use_legacy_opmath")` even for just a string comparison

Reply:
> Got them! Now all tests pass on CI as well 🔥 🔥 🔥
> 
> I'll let you decide if we want to double the test or not with `@pytest.mark.usefixtures("use_legacy_opmath")` even for just a string comparison

I guess for simple things like that we probably won't need it.

Reply:
I'm fine with the way it is currently, since it is not a change in behavior but just the name property

Reply:
That makes sense. The issue is I guess with new opmath enabled by default, this `else ["PauliZ", "PauliZ"]` clause is never tested, because `qml.operation.active_new_opmath()` will always be `True`, but I'm find with not duplicating all these test cases just for a simple string comparison.